examples/ppo/README.md

```diff
@@ -0,0 +1,12 @@
+# Proximal Policy Optimization
```

Comment:
In accordance with #231, could you also add
- owners
- link to tensorboard.dev trace

It would be great having some detailed installation/running instructions (see the WMT example for a good reference).

Reply:
https://github.com/google/flax/pull/470/commits/57dd0a37e81bc582a531254f570befb2cd2a28e3 includes specifying the owners. More detailed running instructions are also included. Regarding the TPU instructions, I decided to link to WMT example to avoid duplication of nearly identical and long text (of course could change that if there's a need to have TPU instructions in every example without linking).

Link to tensorboard.dev trace is going to follow very soon (most likely tomorrow).